- Animate buttons in and out instead of setting their visibility.

shalskar opened the pull request:
This PR will animate the buttons in instead of setting their visibility directly.

zawadz88 commented:
Hi! Thanks for the changes. I run it on a device and the transition looks much smoother!
I've noticed a few glitches though.

1. When the StepperLayout gets created and displayed for the first time there's BACK button fadeout animation which shouldn't really happen.
2. If the adapter contains a single item only there's also a glitch where we see the NEXT button for a while which is then changed to COMPLETE.

zawadz88 commented on material-stepper/src/main/java/com/stepstone/stepper/util/AnimationUtil.java:
```diff
@@ -0,0 +1,39 @@
+import android.support.annotation.NonNull;
+import android.view.View;
+
+public class AnimationUtil {
```
Please add some JavaDoc in this class :) Also `animateView (..)`'s name might be a bit to vague since this method only fades views in and out?

shalskar commented:
Hey, I've updated the PR with the changes you've requested. I believe those issues should be resolved. I've changed the way the method works slightly, which should make it a bit easier to understand. Let me know if you'd like to change anything!

zawadz88 commented:
It looks amazing now :) Please squash and merge ;) Thanks for the PR!
